### PR TITLE
Add Utility Umbrella

### PR DESCRIPTION
**Pull request by KrisXV**
Needs Tested: #6121

### CODE REVIEW

**Comment by urkerab** [comment]
Does Magic Room block Utility Umbrella?

File: data/abilities.js [hunk 1]
```diff
@@ -382,8 +382,10 @@ let BattleAbilities = {
 		num: 167,
 	},
 	"chlorophyll": {
+		desc: "If Sunny Day is active and this Pokemon is not holding Utility Umbrella, this Pokemon's Speed is doubled.",
 		shortDesc: "If Sunny Day is active, this Pokemon's Speed is doubled.",
-		onModifySpe(spe) {
+		onModifySpe(spe, pokemon) {
+			if (pokemon.item === "utilityumbrella") return;
```
Comment (Zarel) [inline data/abilities.js]: These should all be `hasItem`

**Comment by Zarel** [comment]
PR your fixes; this is too high-prio to wait for merge.

**Comment by Marty-D** [comment]
Override the descriptions in Gen 7!!!

**Pull request merged**